llvm/test/Transforms/InstCombine/freeze.ll

```diff
@@ -153,6 +153,19 @@ define i32 @early_freeze_test4(i32 %v1) {
   ret i32 %v2.fr
 }
 
+define i32 @assume(i32 %a, i32 %b) {
+; CHECK-LABEL: @assume(
+; CHECK-NEXT:    call void @llvm.assume(i1 true) [ "noundef"(i32 [[A:%.*]]) ]
+; CHECK-NEXT:    [[ADD_FR:%.*]] = freeze i32 [[ADD:%.*]]
```

Comment:
nit: The name for the freeze input in the CHECK line for this line looks a little odd. I assume `[[ADD:%.*]]` is really referring to `%b`? Can you rename the variable to `[[B:%.*]]`? Thanks!

Reply:
good spot! That'll teach me to blindly trust update_test_checks.py.